packages/apputils/src/dialog.tsx

```diff
@@ -659,7 +677,7 @@ export namespace Dialog {
       title: Header,
       reject: () => void,
       options: Partial<Dialog.IOptions<T>>
-    ): Widget;
+    ): Promise<ReactWidget>;
```

Comment:
I am not a fan of those API changes. Could you try using that API for testing: https://reactjs.org/docs/test-utils.html#act

Reply:
I reverted it BTW, I forgot to mention it. And I found another way to make the tests pass for the Dialog.